tests/foreman/cli/test_remoteexecution.py

```diff
@@ -531,18 +532,22 @@ def test_positive_run_receptor_installer(self, default_sat):
             in ["succeeded", "failed"],
             timeout="1500s",
         )
+
+        result = JobInvocation.get_output({'id': invocation_id, 'host': default_sat.hostname})
+        logger.debug(f"Invocation output>>\n{result}\n<<End of invocation output")
+        # if installation fails, it's often due to missing rhscl repo -> print enabled repos
+        repolist = default_sat.execute('yum repolist')
+        logger.debug(f"Repolist>>\n{repolist}\n<<End of repolist")
+
         assert entities.JobInvocation(id=invocation_id).read().status == 0
-        result = ' '.join(
-            JobInvocation.get_output({'id': invocation_id, 'host': default_sat.hostname})
-        )
         assert 'project-receptor.satellite_receptor_installer' in result
         assert 'Exit status: 0' in result
         # check that there is one receptor conf file and it's only readable
         # by the receptor user and root
         result = default_sat.execute('stat /etc/receptor/*/receptor.conf --format "%a:%U"')
-        assert result.stdout == '400:foreman-proxy'
+        assert result.stdout.strip() == '400:foreman-proxy'
```

Comment:
Can we do this like `"400:foreman-proxy" in result.stdout`?

Reply:
That's what I initially wanted to do but that would also match a lot of unexpected stuff like `69400:foreman-proxy-nonexistent-user LASAGNA WITH SPAM` so I opted to do it this way.

Reply:
@lhellebr if there are more strings in `result.stdout` then this will never pass with `==` operation. Maybe a Regex will help here!